v2/kr/data.go

```diff
@@ -0,0 +1,1232 @@
+package kr
```

Comment:
Was this generated somehow? What was the source? Is there some way to calculate this at runtime rather than have this huge file?

Reply:
I'll find a way.

some of data from..
https://www.kasi.re.kr/kor/index
(Korea Astronomical Research Institute)

but i can't found this meta data today.

---

and chinese lunar calendar data depends on https://github.com/godcong/chronos/blob/master/data.go
I haven't verified it.